src/core/tables/xlsx/xlsxUtils.ts

```diff
@@ -0,0 +1,7 @@
+export function escapeXmlFunctionChars(input: string): string {
+    return input.replace(/&/g, "&amp;")
```

Comment:
Maybe try to find a way to replace the chars at one go instead of calling replace multiple times. Each call goes through the entire string and there can be a lot of long strings in the input table.

Reply:
Replace calls reduced to one in new iteration